package/MDAnalysis/lib/_cutil.pyx

```diff
@@ -235,17 +237,6 @@ def make_whole(atomgroup, reference_atom=None, inplace=True):
             ortho = False
 
     if ortho:
-        # If atomgroup is already unwrapped, bail out
-        is_unwrapped = True
-        for i in range(1, natoms):
-            for j in range(3):
-                if fabs(oldpos[i, j] - oldpos[0, j]) >= half_box[j]:
-                    is_unwrapped = False
-                    break
-            if not is_unwrapped:
-                break
-        if is_unwrapped:
-            return np.array(oldpos)
```

Comment:
I added this shortcut in PR #2189 (see [here](https://github.com/MDAnalysis/mdanalysis/pull/2189/files#diff-d1e4be9d5761ce14019f7d96bbac42eaL213)) because it made `make_whole()` substantially faster (see my [comment](https://github.com/MDAnalysis/mdanalysis/pull/2189#discussion_r252387721) for timings).
I'd only throw that out if there really is no better way of fixing the small box problem.